packages/react-components/package.json

```diff
@@ -22,9 +22,12 @@
     "start": "yarn storybook",
     "docs": "api-extractor run --config=config/api-extractor.local.json --local",
     "build:local": "tsc -p ./tsconfig.lib.json --module esnext --emitDeclarationOnly && node ../../scripts/typescript/normalize-import --output ./dist/packages/react-components/src && yarn docs",
-    "storybook": "start-storybook --port 3000 -s ./public",
+    "prestorybook": "yarn prepare-storybook-local-build",
+    "storybook": "start-storybook --port 3000 -s ./public --no-manager-cache",
+    "prestorybook:docs": "yarn prepare-storybook-local-build",
     "storybook:docs": "start-storybook --port 3000 -s ./public --docs --no-manager-cache",
     "build-storybook": "build-storybook -s ./public -o ./dist/storybook --docs",
+    "prepare-storybook-local-build": "lage build --to @fluentui/babel-make-styles @fluentui/react-storybook-addon",
```

Comment:
Are you going to do the same for all other packages? As the problem remains for them, too:

- `yarn workspace @fluentui/babel-make-styles clean`
- `yarn workspace @fluentui/react-image start --no-manager-cache`

⬇️⬇️⬇️

![image](https://user-images.githubusercontent.com/14183168/149005270-0acc30b1-7829-423d-9feb-235c9a99a6cb.png)

Reply:
hmm not really. I wanted to unblock public facing, which is react-component suite. but probably you're right to have this everywhere to not add additional DX confustion - "it works in react-components but not per package waaat"

I'll update generator and do the migration in follow up PR.

Reply:
https://github.com/microsoft/fluentui/pull/21265
https://github.com/microsoft/fluentui/pull/21267